packages/puppeteer-core/src/common/DownloadBehavior.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2024 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export type DownloadPolicy = 'deny' | 'allow' | 'allowAndName' | 'default';
```

Comment:
I think this one also needs a `@public` annotation and a re-export in packages/puppeteer-core/src/common/common.ts so that docs and types are fully generated.

Reply:
Good catch! fixed it, exported in common.ts and ran `npm run docs` once again :)